backend/src/openarchiefbeheer/destruction/api/serializers.py

```diff
@@ -279,6 +285,14 @@ def update(
 
             instance.add_items(zaken)
 
+        if add_data:
+            zaken = self._get_zaken(zaak_filters, add_data or [], bulk_select)
```

Comment:
```suggestion
            zaken = self._get_zaken(zaak_filters, add_data, bulk_select)
```
Because of the `if add_data`, add_data will always be truthy.

Reply:
Yeah, it seems that the (PyCharm) type checker has issues with this one so I silenced it using this quick and dirty hack, I'm happy to remove it but chances are that we get some typing issues due to another problem?